spec/ddtrace/contrib/rspec/instrumentation_spec.rb

```diff
@@ -5,36 +5,35 @@
 require 'ddtrace'
 
 RSpec.describe 'RSpec hooks' do
-  extend ConfigurationHelpers
-
   let(:configuration_options) { {} }
 
-  around do |example|
-    old_configuration = ::RSpec.configuration
-    ::RSpec.configuration = ::RSpec::Core::Configuration.new
-
+  before do
     Datadog.configure do |c|
       c.use :rspec, configuration_options
     end
-    example.run
+  end
+
+  # Yields to a block in a new RSpec global context. All RSpec
+  # test configuration and execution should be wrapped in this method.
+  def with_new_rspec_environment
```

Comment:
Just to clarify, this is for the RSpec integration, not the RSpec test suite as we use it, correct?

Reply:
This specific one yes. This specific change was extracted to this PR here, which has a better description of this specific change: https://github.com/DataDog/dd-trace-rb/pull/1420